docs/concepts/custom-ui-headless-api.mdx

```diff
@@ -0,0 +1,662 @@
+---
+id: custom-ui-headless-api
+title: Integration Basics
+sidebar_label: Integration Basics
+toc_max_heading_level: 4
+---
+
+```mdx-code-block
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import CodeBlock from "@theme/CodeBlock"
+```
+
+Ory provides headless APIs for ease of integration. This ensures that Ory provides compatibility with many software stacks across
+different programming languages.
+
+This documents' focus is to provide concrete examples on custom UI integration. Flows are the fundamental of Ory's self-service
+APIs and are used to accomplish a variety of goals, such as `login`, `registration`, `settings`, `verification` and `recovery`.
+
+The self-service `flows` are standardized so that it follows the same steps regardless of which `flow` is integrated.
+
+A general overview of the flow can be summed up in two steps:
+
+1. Get or create the `flow` data, e.g. login, registration etc.
+2. Submit the `flow` with user data (e.g. username and password)
+
+For more information on the concepts, take a look at the [Self-service flow](../kratos/self-service) documentation.
+
+## Creating a flow
```

Comment:
In the end this section does not even explain how to create a flow, but only what special params are there.